specification/containerservice/Fleet.Management/fleetnamespace.tsp

```diff
@@ -159,13 +373,32 @@ model FleetManagedNamespaceStatus {
 @doc("A fleet managed namespace.")
 @added(Versions.v2025_03_02_naar)
 model FleetManagedNamespaceProperties {
```

Comment:
Could you package all the single cluster namespace properties into one data structure?



FleetManagedNamespaceProperties {
    ManagedNamespaceProperties {
      labels;
      annotations;
      defaultResourceQuota
      defaultNetworkPolicy
      adoptionPolicy
      deletePolicy
    }
}

Reply:
@circy9 we did consider this, and on the one hand, we agree with keeping everything we inherit from AKS in a single struct.  The thing I don't like about that is that in our API, adoptionPolicy and deletePolicy have an effect wider than ManagedNamespaceProperties.  They govern the application of the AKS bits **and also** the CRP, for example.

Another possibility would be to put everything except adoptionPolicy and deletePolicy into ManagedNamespaceProperties.  How would you feel about that?

Reply:
Yes, I do.

Reply:
`ManagedNamespaceProperties` can't be used because we renamed `FleetManagedNamespace` to it.  I have instead created a new section called `NamespaceProperties`.